### PR TITLE
Modify rule S3985: LaYC format

### PR DESCRIPTION
**Pull request by maksim-grebeniuk-sonarsource**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

File: rules/S3985/rule.adoc [hunk 1]
```diff
@@ -1,16 +1,31 @@
...
 
-[source,text]
+[source,java,diff-id=1,diff-type=noncompliant]
 ----
-public class Foo 
+public class TopLevel
 {
-  ...
-  private class MyUnusedPrivateClass {...} // Noncompliant
+  private class Nested {...} // Noncompliant: Bar is never used
```
Comment (guillaume-dequenne-sonarsource) [inline rules/S3985/rule.adoc:15]: What is `Bar` here? Did you mean `Nested`?

File: rules/S3985/rule.adoc [hunk 2]
```diff
@@ -1,16 +1,31 @@
...
+
+[source,java,diff-id=1,diff-type=compliant]
+----
+public class TopLevel
+{
+  void doSomething() {
+    Nested a = new Nested();
+    ...
+  }
+  private class Nested {...} // Noncompliant
```
Comment (guillaume-dequenne-sonarsource) [inline rules/S3985/rule.adoc:29]: I guess we don't want a `Noncompliant comment` here?

File: rules/S3985/python/rule.adoc [hunk 3]
```diff
@@ -10,23 +12,25 @@ Python has no real private classes. Every class is accessible. There are however
...
 ----
 
-=== Compliant solution
+==== Compliant solution
 
-[source,python]
+[source,python,diff-id=1,diff-type=compliant]
 ----
-class Compliant:
+class TopLevelClass:
```
Comment (guillaume-dequenne-sonarsource) [inline rules/S3985/python/rule.adoc:33]: A bit below, there is some reference to the `Compliant` class that still need to be updated.

**Review by guillaume-dequenne-sonarsource — CHANGES_REQUESTED** [review]
Some small comments.

**Review by guillaume-dequenne-sonarsource — APPROVED** [review]
LGTM

**Review by jean-jimbo-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**